client/src/pages/chatlink/index.js

```diff
@@ -1,14 +1,19 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import './style.css';
 import { renderGoogleReCaptcha, getCaptchaInstance } from './captcha';
 import { getLink } from '../../service';
 import Button from '../../components/Button';
 import LinkDisplay from '../../components/LinkDisplay/index.js';
+import { ThemeContext } from '../../ThemeContext.js';
+import styles from '../../Style.module.css';
+import stylesLocal from './Style.module.css';
```

Comment:
Why `stylesLocal` instead of `styles`?
Also, duplicate import.

Reply:
I thought for someone making changes in the css it would be easier to know where the specific styles associated with its class are. 
Duplicate import?

Reply:
But they both are same file `Style.module.css`